### PR TITLE
os/swap: update instructions

### PR DESCRIPTION
**Pull request by euank**
The previous ones didn't use systemd `.swap` units, which are the
appropriate tool for this.

Note: do not merge until I've tested this, this is untested.

cc @dgonyeo @zbwright as some possible reviewers

### CODE REVIEW

File: os/adding-swap.md [hunk 1]
```diff
@@ -2,63 +2,61 @@
...
 [Install]
 WantedBy=multi-user.target
 ```
 
-**Note** Ensure the block device containing your swapfile is configured to auto-mount on boot.
-
 ### Enable the unit and start using swap
 
-The following command enables and starts the new `swap` service.
+After creating the unit, it can be enabled. Optionally, the `swappiness` value can be modified if your system has sufficient memory for most tasks.
```
Comment (zbwright) [inline os/adding-swap.md]: delete 'Optionally' - Use `systemctl` to enable the unit once created. The `swappiness` value may be modified if the system has sufficient memory.

File: os/adding-swap.md [hunk 2]
```diff
@@ -2,63 +2,61 @@
...
-The following command enables and starts the new `swap` service.
+Use `systemctl` to enable the unit once created. The `swappiness` value may be modified if desired.
 
-```
-$ systemctl enable --now /etc/systemd/system/swap.service
+```sh
+$ systemctl enable --now var-vm-swapfile1.swap
+# Optionally
+$ echo 'vm.swappiness=10' | sudo tee /etc/sysctl.d/80-swappiness.conf
+$ /usr/lib/systemd/systemd-sysctl
```
Comment (bgilbert) [inline os/adding-swap.md]: Should this be starting the systemd service instead?

File: os/adding-swap.md [hunk 3]
```diff
@@ -2,63 +2,61 @@
...
+systemd provides a specialized `.swap` unit file type which may be used to activate swap. The below example shows how to add a swapfile and activate it using systemd.
 
-### Creating the systemd unit file
+### Creating a swapfile
 
-The following systemd unit creates and enables a service to manage a 1GiB swapfile. It should be written to `/etc/systemd/system/swap.service`.
+The following commands will make a 1GiB file suitable for use as swap.
 
+```sh
+mkdir -p /var/vm
```
Comment (bgilbert) [inline os/adding-swap.md]: All of these commands implicitly assume root.  Do we want to say that?

File: os/adding-swap.md [hunk 4]
```diff
@@ -2,63 +2,61 @@
...
+fallocate -l 1024m /var/vm/swapfile1
```
Comment (bgilbert) [inline os/adding-swap.md]: Missing chmod?

**Comment by zbwright** [comment]
tiny bump @euank - did you want to finish this out?

File: os/adding-swap.md (hunk 3 above)
Comment (euank) [reply os/adding-swap.md]: Added `, as root, ` to the line above. For the others (using systemd, writing a unit) I think we frequently assume root in other docs and it's a fairly clear error / systemd login prompt / whatever if you omit it.

File: os/adding-swap.md (hunk 4 above)
Comment (euank) [reply os/adding-swap.md]: Good catch

**Comment by euank** [comment]
Made the suggested changes and ran through the manual steps once more, they worked on a test aws instance.

**Comment by euank** [comment]
I'm leaving it up to someone from the docs team to merge

**Pull request merged**